docs/stats.md

```diff
@@ -92,121 +95,73 @@ them using the general framework directly, so the versions above should be prefe
-corresponding to each allele count *including* the ancestral allele.
-What does this mean?
-Well, `polarised=False` means that we cannot distinguish between an
-allele count of 6 and an allele count of 4.
-So, *folding* means that we would add our allele that is seen in 6 samples
-to `AFS[4]` instead of `AFS[6]`.
-So, in total, we will add 0.5 to each of `AFS[4]`, `AFS[3]`, and `AFS[1]`.
-This means that the sum of an unpolarised AFS
-will be equal to the total number of alleles that are inherited
-by any of the samples in the tree sequence, divided by two.
-Why one-half? Well, notice that if in fact the mutation that produced the *b*
-allele had instead produced an *a* allele,
-so that the site had only two alleles, with frequencies 7 and 3.
-Then, we would have added 0.5 to `AFS[3]` *twice*.
-
-
-(sec_stats_notes_trait)=
-
-#### Trait correlations
-
-{meth}`~TreeSequence.trait_covariance`, {meth}`~TreeSequence.trait_correlation`, and
-{meth}`~TreeSequence.trait_linear_model` compute correlations and covariances of traits
-(i.e., an arbitrary vector) with allelic state, possibly in the context of a multivariate
-linear model with other covariates (as in GWAS).
-
-
-(sec_stats_notes_f)=
-
-#### Patterson's f statistics
-
-{meth}`~TreeSequence.f4`, {meth}`~TreeSequence.f3`, and {meth}`~TreeSequence.f2`
-are the `f` statistics (also called `F` statistics) introduced in
-[Reich et al (2009)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842210/).
-See the documentation (link below) for the definition,
-and [Peter (2016)](https://www.genetics.org/content/202/4/1485) for readable
-discussion of their use.
-
-
-(sec_stats_notes_y)=
+(sec_stats_examples)=
+
+### Quick examples
+
+```{code-cell} ipython3
+:"tags": ["hide-input"]
+from IPython.display import Markdown
+import msprime
+import numpy as np
+
+demography = msprime.Demography()
+demography.add_population(name="A", initial_size=10_000)
+demography.add_population(name="B", initial_size=10_000)
+demography.set_symmetric_migration_rate(["A", "B"], 0.001)
+ts = msprime.sim_ancestry(
+    samples={"A": 2, "B": 2},
+    sequence_length=1000,
+    demography=demography,
+    recombination_rate=2e-8,
+    random_seed=12)
+ts = msprime.sim_mutations(ts, rate=2e-8, random_seed=12)
+Markdown(
+    f"These examples use a tree sequence of {ts.num_samples} samples "
+    f"in {ts.num_populations} populations, "
+    f"with a sequence length of {int(ts.sequence_length)}. "
+    f"There are {ts.num_trees} trees and "
+    f"{ts.num_sites} variable sites in the tree sequence."
+)
+```
 
-#### Y statistics
-
-{meth}`~TreeSequence.Y3` and {meth}`~TreeSequence.Y2` are the `Y` statistics introduced
-by [Ashander et al (2018)](https://www.biorxiv.org/content/10.1101/354530v1)
-as a three-sample intermediate between diversity/divergence (which are
-pairwise) and Patterson's f statistics (which are four-way).
+#### Basic calling convention
 
+```{code-cell} ipython3
```

Comment:
Is the AFS what we want as the first example? It'd be better to use something simple and more representative - the AFS doesn't look like any of the other stats.

Reply:
True. I put it first because the AFS is the basis for most single site stats. But I think you're right. Maybe segregating sites as a simple one.

Reply:
I switched to diversity, to keep the examples similar.